code/go/0chain.net/blobbercore/blobbergrpc/README.md

```diff
@@ -1,19 +1,42 @@
-# GRPC Migration
+# GRPC Endpoints
+
+## Installation
+
+```
+go install \
+github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway \
+github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2 \
+google.golang.org/protobuf/cmd/protoc-gen-go \
+google.golang.org/grpc/cmd/protoc-gen-go-grpc
+```
+
+Run this command to install all the grpc related binaries required to generate files using grpc.
```

Comment:
@shravanshetty1, last word should be `protoc` instead of `grpc`.

Reply:
@shravanshetty1 Should also add that protoc has to be installed

Reply:
@vivekv96